spirv_cross_c.h

```diff
@@ -257,15 +257,24 @@ typedef enum spvc_msl_platform
 	SPVC_MSL_PLATFORM_MAX_INT = 0x7fffffff
 } spvc_msl_platform;
 
+/* Maps to C++ API. */
+typedef enum spvc_msl_index_type
+{
+	SPVC_MSL_NO_INDEX = 0,
```

Comment:
Enum name should be SPVC_MSL_INDEX_TYPE_NONE, INDEX_TYPE_16, INDEX_TYPE_32 to match the enum name.

Reply:
Done.